RELEASE-NOTES.md

```diff
@@ -31,6 +31,7 @@
   * Currently only supports SQL scripts: `post_deploy: [{sql_script: script.sql}]`
 * Added `snow spcs service execute-job` command, which supports creating and executing a job service in the current schema.
 * Added `snow app events` command to fetch logs and traces from local and customer app installations
+* Added support for project definition file defaults in templates
```

Comment:
Doesn't line up with the PR description. Is this intentional?

Reply:
it was also removed below

Reply:
because my older PR took some time, the release note changes went into the wrong section after the merge. They didn't make it to 2.7 though:
https://github.com/snowflakedb/snowflake-cli/releases

So this is kind of not directly related to the changes below.

Reply:
I missed the deletion, 100% my bad.